woocommerce/woocommerce
# Add launch your store success screen

### Changes proposed in this Pull Request:

Closes #45872:

- Adds success page for Launch Your Store
- Adds currently selected class for `CustomerFeedbackSimpleProps` component

<img width="1400" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/11bb5534-e315-417e-a017-0a52769daa70">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

#### Test success page
1. Create a new JN site with this branch.
3. Enable `launch-your-store` feature flag.
4. Go to WooCommerce > Home > Launch Your Store
2. Click on `Launch Store` and `Launch Store Success`
3. Observe confetti animation after the page is loaded

#### Test copy link
1. Click on `Copy link`
2. Observe the copy button is changed to `Copied!` and then revert after a couple of seconds
3. Paste anywhere and ensure the link is the site's homepage

#### Test visit store link
1. Click on `Visit your store`
2. Observe it opens up the homepage in a new tab

#### Test survey
1. Click on any emoji next to `How was the experience of launching your store?`
2. Observe a textarea labelled `Why do you feel that way?` is shown
3. Type anything into the textarea
4. Click `Cancel`
5. Observe the textarea disappear and emoji is deselected
6. Click on any emoji again
7. Observe the textarea is brought back with previous contents
8. Click on `Send`
9. Observe `🙌 We appreciate your feedback!` is shown
10. Refresh the page
11. Observe survey form (emoji list) is no longer shown

#### Test next steps
1. With no tasks completed, observe the next steps are `Promote your products`, `Provide more ways to pay`, and `Power up your store`
2. Click on `Promote products` on a new tab
3. Observe you're navigated to Marketing screen
4. Install `TikTok for WooCommerce` plugin
5. Refresh success screen
6. Observe the new list is `Provide more ways to pay`, `Power up your store`, and `Build customer relationships`
7. Test around with next steps, it should work according to comments here: 0BWMerqRR1f468EwC7g5tg-fi-925_15371




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
